src/Lesson22/Main.java

```diff
@@ -0,0 +1,56 @@
+package Lesson22;
+
+/*
+  Домашнее задание №22
+   Создать программу для чтения номеров контрактов из файла.
+   Неповторяющиеся номера контрактов следует сохранить в структуру данных
+   ключ-значение, где ключ - это номер контракта, а значение - это
+   объект класса Document со всей контекстной информацией (номер
+   документа, дота создания и другая информация).
+
+   Структуру данных ключ-значение, использавать коллекции Set и Map.
+
+   Файл, откуда ты читаешь информацию содержит несколько полей, разделенных запятой, например
+   Что то типо:
+   Номер документа,название документа,дата создания,автор
+
+   Путь к файлу: src/Lesson22/document_numbers
+ */
+
+import java.io.*;
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        System.out.println("Введите путь к файлу:");
+        Scanner scanner = new Scanner(System.in);
+        String path = scanner.nextLine();
+        Map<String, Document> mapDocument = new HashMap<>();
+
+        File file = new File("src/Lesson22/document_numbers");
+        try (FileReader fr = new FileReader(file);
+        BufferedReader reader = new BufferedReader(fr)) {
```

Comment:
Try to avoid `try-with-resources` construction, at least when you have more than one resource inside. It's better to close it in finally block